app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(line: str) -> None:
```

Comment:
The function name 'copy_file' is descriptive, which is good. Ensure that the input command 'cp' is correct as per the task requirements.